examples_script/napoleon/napoleon.py

```diff
@@ -1,46 +1,23 @@
 #!/usr/bin/env python3
 
 
-import sys
-sys.path.append("../../.")
-from autocards import Autocards
+# import sys
+# sys.path.append("../../.")
+from autocards.autocards import Autocards
 from pathlib import Path
 
 
 prefix = "On Napoléon : "
-file = Path("./napoleon.txt")
+file = Path("/path/to/Autocards/examples_script/napoleon/napoleon.txt")
```

Comment:
I don't understand this change. To me the idea was to make this example script executable without having to set any path or to look at the codebase.